attentive-android-sdk/src/main/java/com/attentive/androidsdk/push/TokenProvider.kt

```diff
@@ -18,21 +18,7 @@ class TokenProvider {
 
     internal suspend fun getToken(context: Context): Result<TokenFetchResult> {
         Timber.d("getToken")
-        var clientWillHandlePushToken = false
-//        AttentiveEventTracker.instance.config?.clientWillHandlePushToken?.let {
-//            clientWillHandlePushToken = it
-//        }
-
-        if (clientWillHandlePushToken.not()) {
-            return getTokenFromFirebase(context)
```

Comment:
This was the only line ever being used, so i removed the rest. It is leftover from the work I was doing when I thought we needed to support multiple firebase projects in one app.